src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const populationCountry = document.querySelectorAll('span');
+const len = document.querySelectorAll('.population').length;
```

Comment:
The variable `len` is correctly selecting elements with the class 'population', but it should be used to select 'span' elements with this class, similar to the correction needed for `populationCountry`.